Snowflake.Data/Core/HttpUtil.cs

```diff
@@ -107,26 +107,25 @@ private HttpClient RegisterNewHttpClientIfNecessary(HttpClientConfig config)
         private HttpMessageHandler setupCustomHttpHandler(HttpClientConfig config)
         {
             HttpMessageHandler httpHandler;
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && SFEnvironment.ClientEnv.IsNetFramework)
+            try
             {
-                httpHandler = new WinHttpHandler()
+                httpHandler = new HttpClientHandler()
                 {
                     // Verify no certificates have been revoked
                     CheckCertificateRevocationList = config.CrlCheckEnabled,
                     // Enforce tls v1.2
                     SslProtocols = SslProtocols.Tls12,
                     AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate,
-                    CookieUsePolicy = CookieUsePolicy.IgnoreCookies
+                    UseCookies = false // Disable cookies
                 };
             }
-            else
+            // special logic for .NET framework 4.7.1 that
+            // CheckCertificateRevocationList and SslProtocols are not supported
+            catch (PlatformNotSupportedException)

```

Comment:
This is needed because CheckCertificateRevocationList and SslProtocols are not implemented in 4.7.1 and could throw PlatformNotSupportedException, in such case removing setting of CheckCertificateRevocationList and SslProtocols could solve the issue.
https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclienthandler.checkcertificaterevocationlist
https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclienthandler.SslProtocols
`.NET Framework 4.7.1 only: This property is not implemented.`

This is introduced by supporting .NET Framework 4.7.1, which was just release in May, 2.0.24.